### DISCUSSION TITLE
Preserve Folder Structure

### DISCUSSION THREAD

**Discussion by ian-barber** [parcel-bundler/parcel#9923] [Q&A]
Is there a way to make Parcel Preserve the Folder Structure rather than putting everything in the root of Dist?

**Comment by jools-r** [comment]
For static files at least, there's [parcel-reporter-static-files-copy](https://www.npmjs.com/package/parcel-reporter-static-files-copy) which works reliably for files that don't need processing. You may find other useful namer plugins in [parcel's plugin search](https://parceljs.org/plugin-browser/?type=%22namer%22&page=0&filter=%22%22&includeOfficial=false), for example perhaps [parcel-namer-preserve-structure](https://www.npmjs.com/package/@mischnic/parcel-namer-preserve-structure)?
